pkg/operator/apis/monitoring/v1/types.go

```diff
@@ -698,7 +706,7 @@ func endpointScrapeConfig(id, projectID, location, cluster string, ep ScrapeEndp
 	scrapeCfg := &promconfig.ScrapeConfig{
 		// Generate a job name to make it easy to track what generated the scrape configuration.
 		// The actual job label attached to its metrics is overwritten via relabeling.
-		JobName:                 fmt.Sprintf("%s/%s", id, &ep.Port),
+		JobName:                 id,
```

Comment:
Unfortunately we have to be more careful here in terms of breaking change, so let's see in https://github.com/GoogleCloudPlatform/prometheus-engine/issues/479 if we want to pursue this and how 👍🏽

Reply:
Yes, I agree.

I'm keeping this PR open as a draft until a better way is found in #479. Then we'll see if this PR can be improved or if it's better to make a new one.

Or, if you think it makes sense to just close it now to get it out of the list, feel free :-)